playwright/README.md

```diff
@@ -0,0 +1,67 @@
+# Playwright Testing ReadMe
+
+A small readMe on how to run the playwright suite and the commands required
+
+## Table of Contents
+
+* [How to set up throttle limiter for testing](#Throttler-Setup)  
+* [How to run playwright](#Running-Playwright)
+    * [Running all tests](#running-all-tests)  
+    * [Running a single test](#running-a-single-test)
+    * [Run a set of tests](#run-a-set-of-test-files)  
+    * [Running tests in heeaded mode](#running-tests-in-headed-mode)
+
+
+## How to set up throttle limiter for testing
+
+When building the test suite it was found that the tests would max out the rate limiter. A change was made to make this a variable which can be set to different values. 
+
+To set this up for testing do the following:
+
+* Make sure you have down docker compose down 
+* Before building a new docker image go into `app.js` and set the `DEFAULT_RATE_LIMIT` variable to 
```

Comment:
I know this sentence isn't done, Just waiting on confirmation of what number we agreed on for default rate limiter to be able to run tests